draft-packages/likert-scale-legacy/docs/LikertScaleLegacy.stories.tsx

```diff
@@ -65,31 +69,82 @@ export const DefaultStory = () => {
         selectedItem={selectedItem}
         onSelect={item => setSelectedItem(item)}
       />
-    </div>
+    </>
   )
 }
-DefaultStory.storyName = "Default"
+DefaultStory.storyName = "Default (Kaizen Demo)"
 
-export const Reversed = () => {
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => {
   const [selectedItem, setSelectedItem] = useState<ScaleItem | null>(null)
-
   return (
-    <div>
-      <div style={{ marginBottom: "40px" }} id={labelId}>
-        <Heading variant="heading-4" color="white">
-          How would you rate this survey?
-        </Heading>
-      </div>
-      <LikertScaleLegacy
-        scale={scale}
-        automationId="123"
-        labelId={labelId} // Intended to match the id of the label
-        selectedItem={selectedItem}
-        onSelect={item => setSelectedItem(item)}
-        reversed
-      />
-    </div>
+    <>
+      <StoryWrapper isReversed={isReversed}>
+        <StoryRowHeader headings={[]} />
+        <StoryWrapper.Row rowTitle="Very Poor">
```

Comment:
Since this component is legacy, it's probably not worth cleaning this up too much (eg. the `labelId` is actually making these all point to the label in the default story, the code snippet isn't nice about the `onSelect` (I've been looking into how it generates docs, but there's no good balance; you either show the raw code or the generated, but not a combination)), but to follow [the sticker sheet in Figma](https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=14473%3A61902) maybe add an empty scale and change the labels.